src/app/client/src/app/modules/learn/components/course-consumption/course-consumption-header/course-consumption-header.component.ts

```diff
@@ -263,6 +263,12 @@ export class CourseConsumptionHeaderComponent implements OnInit, AfterViewInit,
     }
   }
 
+
+  openDiscussionForum() {
+    this.router.navigate(['/discussions']);
```

Comment:
What will happen when we click on browser back after navigating to discussion forum